dll/win32/newdev/newdev.c

```diff
@@ -1,7 +1,7 @@
 /*
  * New device installer (newdev.dll)
  *
- * Copyright 2005-2006 Herv� Poussineau (hpoussin@reactos.org)
+ * Copyright 2005-2006 Hervé Poussineau (hpoussin@reactos.org)
```

Comment:
You might have broken the file encoding here.

Reply:
(Or fixed, can't tell from the github interface)

Reply:
Fixed by converting to utf8